src/cli/generator/config.rs

```diff
@@ -258,8 +260,20 @@ impl Config {
             .collect::<anyhow::Result<Vec<Input<Resolved>>>>()?;
 
         let output = self.output.resolve(parent_dir)?;
+        let secret = if let Some(secret) = self.secret {
+            let template = Mustache::parse(&secret)?;
+            Some(template.render(&reader_context))
+        } else {
+            None
+        };
 
-        Ok(Config { inputs, output, schema: self.schema, preset: self.preset })
+        Ok(Config {
+            inputs,
+            output,
+            schema: self.schema,
+            preset: self.preset,
+            secret,
```

Comment:
Secrets need to be used in the LLM. Read my message — https://github.com/tailcallhq/tailcall/issues/2658